ops/mappings.json

```diff
@@ -610,6 +610,13 @@
         "dao": "base",
         "mapping": "Redeemer",
         "arcVersion": "0.0.1-rc.31"
+      },
+      {
+        "name": "DAOTracker",
+        "contractName": "DAOTracker",
+        "dao": "base",
+        "mapping": "DAOTracker",
+        "arcVersion": "0.0.1-rc.29"
```

Comment:
it should be 0.0.1-rc.31

Reply:
The contract was introduced in rc.29, and the address & ABI haven't changed since, so I think it should be rc.29 no?

Reply:
the private network (ganache) mapping should include all latest version contracts.
(need to set the number to 18 here https://github.com/daostack/subgraph/blob/master/test/sanity.spec.ts#L15 )

Reply:
Updated